layers/+lang/python/funcs.el

```diff
@@ -217,15 +217,14 @@ ROOT-DIR should be the path for the environemnt, `nil' for clean up"
 
 ;; from https://www.snip2code.com/Snippet/127022/Emacs-auto-remove-unused-import-statemen
 (defun spacemacs/python-remove-unused-imports ()
-  "Use Autoflake to remove unused function"
-  "autoflake --remove-all-unused-imports -i unused_imports.py"
+  "Use Autoflake to remove the unused imports:
```

Comment:
the first line of any function documentation should be a complete sentence ends with a period (.).

Reply:
Modified in that latest change.